src/Jackett.Common/Indexers/Definitions/TorrentDosFilmes.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using AngleSharp.Dom;
+using AngleSharp.Html.Parser;
+using Jackett.Common.Extensions;
+using Jackett.Common.Indexers.Definitions.Abstract;
+using Jackett.Common.Models;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using NLog;
+using WebClient = Jackett.Common.Utils.Clients.WebClient;
+using WebRequest = Jackett.Common.Utils.Clients.WebRequest;
+
+namespace Jackett.Common.Indexers.Definitions
+{
+    [ExcludeFromCodeCoverage]
+    public class TorrentDosFilmes : PublicBrazilianIndexerBase
+    {
+        public override string Id => "torrentdosfilmes";
+        public override string Name => "TorrentDosFilmes";
+        public override string SiteLink { get; protected set; } = "https://torrentdosfilmes.site/";
+
+        public override string[] AlternativeSiteLinks { get; protected set; } = {
+            "https://ComandoFilmes.xyz/"
```

Comment:
```suggestion
            "https://torrentdosfilmes.site/",
            "https://ComandoFilmes.xyz/"
```

Reply:
the alternativelinks needs to include the sitelink or it will not show up in the dashboard config